src/main/java/pl/zaprogramujzycie/mma/utils/exceptions/ErrorMessage.java

```diff
@@ -0,0 +1,24 @@
+package pl.zaprogramujzycie.mma.utils.exceptions;
+
+import lombok.Getter;
+
+import java.util.Date;
+
+public class ErrorMessage {
```

Comment:
propnuje zrobic to jako rekord

Reply:
Muszę to jeszcze przetestować, ale jak rzucam wyjątek to podaję mu message i teoretycznie powinno pobierać wiadomości w rodzaju "Medicine doesn't exist", "Available only for the authorised user".

Reply:
Zostawiłam to sobie na później oznaczone jako ToDo, bo fajnie by było mieć jakieś customowe wiadomości dla użytkowników. Ale na razie jest bez wiadomości, działa jak należy i nie będę tego ruszać, bo zmiana Error Message pociąga za sobą dłubanie we wszystkich hendlerach.